src/status_im/translations/en.cljs

```diff
@@ -135,22 +135,22 @@
    :backup-your-seed                     "Backup your Seed Phrase"
 
    ;;seed
-   :your-data-belongs-to-you             "Your Data and Funds belong to you. We can’t help get it back if you loose it"
-   :your-data-belongs-to-you-description "This means Status can’t help you get it back if you lose it. You are in charge of the secuty of all your data"
+   :your-data-belongs-to-you             "If you lose your seed phrase you lose your data and funds."
+   :your-data-belongs-to-you-description "This means Status can’t help you get it back if you lose it. You are in charge of the security of your data."
    :ok-continue                          "Ok, continue"
-   :your-seed-phrase                     "Your Seed Phrase"
-   :your-seed-phrase-description         "This is your seed phrase. You use it to prove that this is your wallet. You only get to see it once! Copy it on paper and keep it in a secure place. You need it when you loose or reinstall your wallet."
+   :your-seed-phrase                     "Your seed phrase"
+   :your-seed-phrase-description         "This is your seed phrase. You use it to prove that this is your wallet. You only get to see it once! Write it on paper and keep it in a secure place. You will need it if you lose or reinstall your wallet."
    :enter-word                           "Enter word"
-   :check-your-seed                      "Check your Seed Phrase"
+   :check-your-seed                      "Check your seed phrase"
    :wrong-word                           "Wrong word"
    :are-you-sure?                        "Are you sure?"
    :are-you-sure-description             "You will not be able to see the whole seed phrase again"
    :you-are-all-set                      "You’re all set!"
-   :you-are-all-set-description          "Now if you loose your phone you can restore your account and wallet using the Seed Phrase and password"
+   :you-are-all-set-description          "Now if you lose your phone you can restore your account and wallet using the seed phrase and password."
    :ok-got-it                            "Ok, got it"
    :backup-seed-phrase                   "Backup Seed Phrase"
    :step-i-of-n                          "Step {{step}} of {{number}}"
-   :word-n-description                   "In order to check if have backed up your seed phrase correctly, enter the word #{{number}} above"
+   :word-n-description                   "In order to check if have backed up your seed phrase correctly, enter the word #{{number}} above."
```

Comment:
Insert "you" : "In order to check if YOU have backed up your seed"

Reply:
👍